api/types/types.go

```diff
@@ -396,13 +396,15 @@ type NetworkResource struct {
 	Name       string                         // Name is the requested name of the network
 	ID         string                         `json:"Id"` // ID uniquely identifies a network on a single machine
 	Created    time.Time                      // Created is the time the network created
-	Scope      string                         // Scope describes the level at which the network exists (e.g. `global` for cluster-wide or `local` for machine level)
+	Scope      string                         // Scope describes the level at which the network exists (e.g. `swarm` for cluster-wide or `local` for machine level)
```

Comment:
We discussed this internally, and the reason we opted for `swarm` is because it is easy to associate to the swarm feature. Also in case of single host networks promoted to swarm scope, `global` would not be correct because the network does is not a multihost network. It  just indicates the network is usable within swarm.